### ISSUE TITLE
Remove kinesis_max_throttled_wait_ms config from AWS UI setup flow

### ISSUE THREAD

**Issue by danotorrey** [Graylog2/graylog-plugin-integrations#156]
Remove the `kinesis_max_throttled_wait_ms` from the AWS CloudWatch UI save flow. This field is no longer needed. The Kinesis Client Library in Java no longer requires this value for throttling. This was removed from the backend in https://github.com/Graylog2/graylog-plugin-integrations/commit/241e7f24d463a30ad711a3ed345d4a123b5a2dcb. #151 must be merged before this change can be effective. I created this issue now, so I don't lose track of it.

**Comment by danotorrey** [comment]
Assigning this to me for now, so I don't lose track of it.

**Comment by danotorrey** [comment]
This change can be implemented now.

**Comment by danotorrey** [comment]
Make sure to remove the ignore annotation added in https://github.com/Graylog2/graylog-plugin-integrations/commit/b2f81e999696ca9cd1f78b5d846351ac1a17caa5.
